influxql/iterator_test.go

```diff
@@ -867,6 +867,50 @@ func TestIteratorOptions_DerivativeInterval_Call(t *testing.T) {
 	}
 }
 
+func TestIteratorOptions_ElapsedInterval_Default(t *testing.T) {
+	opt := influxql.IteratorOptions{}
+	expected := influxql.Interval{Duration: time.Second}
+	actual := opt.ElapsedInterval()
+	if actual != expected {
+		t.Errorf("expected elapsed interval to be %v, got %v", expected, actual)
+	}
+}
+
+func TestIteratorOptions_ElapsedInterval_GroupBy(t *testing.T) {
+	opt := influxql.IteratorOptions{
+		Interval: influxql.Interval{
+			Duration: 10,
+			Offset:   2,
+		},
+	}
+	expected := influxql.Interval{Duration: 10}
```

Comment:
This doesn't seem correct. `ElapsedInterval()` is supposed to ignore `GROUP BY time(...)`, right? It looks like this is checking that the ElapsedInterval changes if there's a `GROUP BY`.